src/nuklear_font.c

```diff
@@ -2318,11 +303,13 @@ nk_font_bake(struct nk_font_baker *baker, void *image_memory, int width, int hei
         it = config_iter;
         do {const struct nk_font_config *cfg = it;
             struct nk_font_bake_data *tmp = &baker->build[input_i++];
-            nk_tt_PackSetOversampling(&baker->spc, cfg->oversample_h, cfg->oversample_v);
-            nk_tt_PackFontRangesRenderIntoRects(&baker->spc, &tmp->info, tmp->ranges,
-                (int)tmp->range_count, tmp->rects, &baker->alloc);
+            stbtt_PackSetOversampling(&baker->spc, cfg->oversample_h, cfg->oversample_v);
+			// TODO:
+            // was: stbtt_PackFontRangesRenderIntoRects(&baker->spc, &tmp->info, tmp->ranges, (int)tmp->range_count, tmp->rects, &baker->alloc);
+            stbtt_PackFontRangesRenderIntoRects(&baker->spc, &tmp->info, tmp->ranges, (int)tmp->range_count, tmp->rects);
```

Comment:
The stb allocator should use the Nuklear allocator. The following is from `stb_truetype.h`:

```
   // #define your own functions "STBTT_malloc" / "STBTT_free" to avoid malloc.h
￼   #ifndef STBTT_malloc
￼   #include <stdlib.h>
￼   #define STBTT_malloc(x,u)  ((void)(u),malloc(x))
￼   #define STBTT_free(x,u)    ((void)(u),free(x))
￼   #endif
```

Reply:
How do I bump the Nuklear version?

Reply:
See https://github.com/Immediate-Mode-UI/Nuklear/blob/master/src/CHANGELOG